module/data/actor/templates/sentient.mjs

```diff
@@ -126,6 +126,11 @@ export default class SentientTemplate extends CommonTemplate {
                         integer: true,
                         label: "ED.Actor.Characteristics.woundThreshold"
                     } ),
+                    maxWounds: new fields.NumberField( {
```

Comment:
Just for me to understand: this means that everything sentient (including PCs, dragons, horrors, spirits, etc) can be a mob? Or should that maybe be a special SchemaField or MixIn that only applies to certain subtypes?

Reply:
everyone except for characters... i was struggling with this